src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java

```diff
@@ -41,6 +41,8 @@ public static PcqAnswerResponse getPcqResponseFromDomain(ProtectedCharacteristic
         PcqAnswerResponse answerResponse = new PcqAnswerResponse();
 
         answerResponse.setPcqId(protectedCharacteristics.getPcqId());
+        answerResponse.setDcnNumber(protectedCharacteristics.getDcnNumber());
```

Comment:
Using @Builder annotation on beans is good for this as you simple do
PcqAnswerResponse answerResponse = new PcqAnswerResponse.Builder
    .setPcqId(protectedCharacteristics.getPcqId())
    .setDcnNumber(protectedCharacteristics.getDcnNumber())
    .build();